### PR TITLE
Add auto-generated tests and travis config to Dart clients

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`. If contributing template-only or documentation-only changes which will change sample output, be sure to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) first.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.1.x`, `5.0.x`. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Redo https://github.com/OpenAPITools/openapi-generator/pull/2909

### CODE REVIEW

**Comment by wing328** [comment]
cc @ircecho (2017/07) @swipesight (2018/09) @jaumard (2018/09)

File: modules/openapi-generator/src/main/resources/dart-jaguar/model_test.mustache [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{{#models}}
+{{#model}}
+import 'package:{{pubName}}/api.dart';
+import 'package:test/test.dart';
+
+// tests for {{classname}}
+void main() {
+  var instance = new Pet();
```
Comment (ircecho) [inline modules/openapi-generator/src/main/resources/dart-jaguar/model_test.mustache]: No longer builds. This should be
```dart
var instance = new {{classname}}();

Comment (wing328) [reply modules/openapi-generator/src/main/resources/dart-jaguar/model_test.mustache]: Good catch. I'll fix it.

Comment (wing328) [reply modules/openapi-generator/src/main/resources/dart-jaguar/model_test.mustache]: Filed https://github.com/OpenAPITools/openapi-generator/pull/3014

**Pull request merged**